contrib/checkpoint/testdata/container_sleep.json

```diff
@@ -0,0 +1,49 @@
+{
+	"metadata": {
+		"name": "podsandbox-sleep"
+	},
+	"image": {
+		"image": "quay.io/adrianreber/counter:latest"
+	},
+	"command": [
+		  "/bin/bash"
+	],
+	"args": [
+		"-c",
+		"while true; do echo -n 'hello: '; date; sleep 0.5;done"
+
+	],
+	"working_dir": "/",
+	"envs": [
+		{
+			"key": "PATH",
+			"value": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+		},
+		{
+			"key": "GLIBC_TUNABLES",
+			"value": "glibc.pthread.rseq=0"
```

Comment:
IIRC, this was a temporarily workaround for the problem with restartable sequences (https://github.com/checkpoint-restore/criu/issues/1885). Is it still needed?

Reply:
It doesn't hurt to be here and ensures it works on older kernels without the ptrace req interface.